committee/models/committee.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from conference.models.event import Event
+
+
+class Committee(models.Model):
+    title = models.CharField(
+        max_length=1024,
+        help_text=_("Review Committee"),
+        verbose_name=_("Review Committee"),
+        null=True,
+    )
+
+    event = models.ForeignKey(
+        "conference.Event",
+        on_delete=models.PROTECT,
+        verbose_name=_("Event"),
+    )
+
+    def get_event(self):
+        event = Event.objects.filter(year=settings.CURRENT_EVENT).only("name").first()
+        if event.name:
+            return event.name
+        return "Untitled Event"
+
+    def __repr__(self):
+        return f"Review Committee of {self.get_event}"
```

Comment:
if you want to use it like this, you have to mark get_event as a property abouve

Reply:
btw, you also need __str__ here, not __repr__ Maybe __repr__ is also useful somewhere, but for returning name somewhere else, for example in comittee_member, you need __str__ Please fix this too.